app/controllers/application_controller.rb

```diff
@@ -1,3 +1,5 @@
 class ApplicationController < ActionController::Base
-  protect_from_forgery with: :exception
+  protect_from_forgery with: :null_session
```

Comment:
This will not protect you from fraudulent request, manage the exception or create your own forgery strategy